File: app/src/main/java/com/organizobra_mobile/ui/report/ReportModel.java
package com.organizobra_mobile.ui.report;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ReportModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public ReportModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is people fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/organizobra_mobile/ui/report/ReportModel.kt
package com.organizobra_mobile.ui.report

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ReportModel : ViewModel() {
    private val mText = MutableLiveData<String>()

    init {
        mText.value = "This is people fragment"
    }

    val text: LiveData<String>
        get() = mText
}